partner_gender/README.rst

```diff
@@ -0,0 +1,59 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Partner Gender
+==============
+
+This module was written to extend the functionality of `base_contact`_ to
```

Comment:
Agreed, we should be upfront about that dependency. They are begining to be more of a family of modules.